lib/bracket-matcher.coffee

```diff
@@ -218,3 +224,6 @@ class BracketMatcher
 
   unsubscribe: ->
     @subscriptions.dispose()
+
+  getScopedSetting: (key) ->
+    atom.config.get(key, scope: @editor.getLastCursor().getScopeDescriptor())
```

Comment:
What about using `TextEditor.prototype.getRootScopeDescriptor` here? I feel like it might be better, because we don't really care about per-line scopes in this case it would seem. Thoughts? :thought_balloon: